hieradata/integration.yaml

```diff
@@ -31,6 +31,9 @@ govuk::apps::whitehall::basic_auth_credentials: "%{hiera('http_username')}:%{hie
 govuk::apps::whitehall::highlight_words_to_avoid: true
 govuk::apps::whitehall::enable_tagging_to_new_taxonomy: "yes"
 
+govuk_beat::hosts:
```

Comment:
I'm not sure why we shouldn't just override govuk_beat::hosts in the credentials repo, does the logstash_host variable already exist?